src/app/zones/views/CreateView.vue

```diff
@@ -284,28 +284,29 @@
                 </div>
 
                 <div class="form-section">
-                  <div class="form-section__header">
-                    <h2 class="form-section-title">
-                      {{ t('zones.form.section.scanner.title') }}
-                    </h2>
-
-                    <p v-if="t('zones.form.section.scanner.description') !== ' '">
-                      {{ t('zones.form.section.scanner.description') }}
-                    </p>
-                  </div>
-
-                  <div class="form-section__content">
-                    <DataSource
-                      v-slot="{ data, error: scanError }: ZoneOverviewSource"
-                      :src="`/zone-cps/online/${name}?no-cache=${Date.now()}`"
-                      @change="success"
+                  <DataSource
+                    v-slot="{ data, error: scanError }: ZoneOverviewSource"
+                    :src="`/zone-cps/online/${name}?no-cache=${now}`"
```

Comment:
I moved the DataSOurce up a level so I can stop showing the 'title' if there is an error (the title and the body copy currently say "Waiting etc etc". If there is an error we should no longer say "Waiting" anywhere.

I also moved the poor mans cache bust date to be the same throughout the lifetime of the CreateView route. There'll be a better way to define cache busting at a later date.